sprints/20201110-apple-silicon-fenix-shipit/sprint-review.md

```diff
@@ -0,0 +1,44 @@
+# Sprint review - Apple Silicon and Fenix on Shipit, October 26, 2020 - November 6, 2020
+
+Or, "There's no perfect time to start sprints".
+
+While members of the Releng team had participated in scrum at other companies, a number of us were new to agile, and none of us had Scrum Master experience. But we knew we needed to change how we worked with our reduced team size and combined team structure.
+
+A number of team members had PTO booked, meaning there was no perfect two-week span for our first sprint before US Thanksgiving week. We decided that starting the sprints on an imperfect week was preferable over waiting for some mythical perfect time.
+
+Aki had the most background from the [Taskcluster team's sprints](https://github.com/taskcluster/scrum/) and planned to help keep things on track in week 2, after coming back from PTO. However, life happened, and he was out both weeks, on bereavement leave.
+
+We tried two parallel sprints, one per epic, and some non-sprinting team members to handle interrupts and releaseduty. We largely didn't use JIRA to track things. There was no Product Owner on the Apple Silicon sprint. Mihai was understandably unable to handle both his new management role and the Scrum Master role, so we were Scrum-Master-less as well.
```

Comment:
Oups, mea culpa again for this. O_O

Reply:
No worries. Trying to just describe what happened; no blame here.